docs-conceptual/azps-2.0.0/migrate-az-1.0.0.md

```diff
@@ -42,146 +47,181 @@ This document describes the changes between the 6.x versions of AzureRM and Az v
 #Requires -Modules AzureRM.Storage
 Get-AzureRmStorageAccount | Get-AzureStorageContainer | Get-AzureStorageBlob
 ```
 
-Could be run with minimal changes using ```Enable-AzureRmAlias```:
-```powershell
+Can be run with minimal changes using `Enable-AzureRmAlias`:
+
+```azurepowershell-interactive
 #Requires -Modules Az.Storage
-Enable-AzureRmAlias
+Enable-AzureRmAlias -Scope Process
 Get-AzureRmStorageAccount | Get-AzureStorageContainer | Get-AzureStorageBlob
 ```
 
-Running ```Enable-AzureRmAlias -Scope CurrentUser``` will enable the aliases for all powershell sessions you open, so that after executing this cmdlet, a script like this would not need to be changed at all:
-```powershell
+Running `Enable-AzureRmAlias -Scope CurrentUser` will enable the aliases for all PowerShell sessions you open, so that after executing this cmdlet, a script like this would not need to be changed at all:
+
+```azurepowershell-interactive
 Get-AzureRmStorageAccount | Get-AzureStorageContainer | Get-AzureStorageBlob
 ```
 
-For complete details on the usage of the alias cmdlets, execute ```Get-Help -Online Enable-AzureRmAlias``` from the powershell prompt.
+For complete details on the usage of the alias cmdlets, see the [Enable-AzureRmAlias reference](/powershell/module/az.accounts/enable-azurermalias).
 
-```Disable-AzureRmAlias``` removes AzureRM cmdlet aliases created by ```Enable-AzureRmAlias```.  For complete details, execute ```Get-Help -Online Disable-AzureRmAlias``` from the powershell prompt.
+When you're ready to disable aliases, `Disable-AzureRmAlias` removes the created aliases. For complete details,
+see the [Disable-AzureRmAlias reference](/powershell/module/az.accounts/disable-azurermalias).
+
+> [!IMPORTANT]
+> When disabling aliases, make sure that they are disabled for _all_ scopes which had aliases enabled.
 
 ### Module Name Changes
-- The module names have changed from `AzureRM.*` to `Az.*`, except for the following modules:
-```
-AzureRM.Profile                       -> Az.Accounts
-Azure.AnalysisServices                -> Az.AnalysisServices
-AzureRM.Consumption                   -> Az.Billing
-AzureRM.UsageAggregates               -> Az.Billing
-AzureRM.DataFactories                 -> Az.DataFactory
-AzureRM.DataFactoryV2                 -> Az.DataFactory
-AzureRM.MachineLearningCompute        -> Az.MachineLearning
-AzureRM.Insights                      -> Az.Monitor
-AzureRM.RecoveryServices.Backup       -> Az.RecoveryServices
-AzureRM.RecoveryServices.SiteRecovery -> Az.RecoveryServices
-AzureRM.Tags                          -> Az.Resources
-Azure.Storage                         -> Az.Storage
-```
 
-The changes in module names mean that any script that uses ```#Requires``` or ```Import-Module``` to load specific modules will need to be changed to use the new module instead.
+The module names have changed from `AzureRM.*` to `Az.*`, except for the following modules:
+
+| AzureRM module | Az module | Cmdlet suffix changed? |
```

Comment:
It's not clear to me what a "suffix" is in this context. Do you mean the cmdlet Noun? Suffix is not a term that has been used in PowerShell. So this should be changed or defined.

Reply:
This was meant to represent things like where the module name was part of a cmdlet name, and whether or not that changed along with the module name (most cmdlets didn't) but you're right that it's too confusing. Removing.